FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Callback/DeviceBindingCallbackTests.swift

```diff
@@ -0,0 +1,2079 @@
+// 
+//  DeviceBindingCallbackTests.swift
+//  FRAuthTests
+//
+//  Copyright (c) 2022 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import XCTest
+import JOSESwift
+@testable import FRAuth
+
+class DeviceBindingCallbackTests: FRAuthBaseTest {
+    
+    func test_01_basic_init() {
+        let jsonStr = """
```

Comment:
We have most of the content of this `jsonStr` are present in all tests. I would suggest move this to a function with parameters to improve the reusability.

Reply:
@rodrigoareis extracted to a helper method with bunch of parameters with default values. Way less code.